developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -209,3 +328,51 @@ File properties can be requested by adding a :code:`<d:prop/>` element to the re
 When listing favorites, the request will find all favorites in the folder recursively, all favorites for a user can be found by sending the request to :code:`remote.php/dav/files/user`
 
 .. _rfc4918: https://tools.ietf.org/html/rfc4918
+
+
+Special Headers
+---------------
+
+Request Headers
+^^^^^^^^^^^^^^^
+
+You can set some special headers that Nextcloud will interpret.
+
++-----------------+-----------------------------------------------------------------+----------------------------------+
+|     Header      |                           Description                           |             Example              |
++=================+=================================================================+==================================+
+| X-OC-MTime      | | Allow to specify a mtime.                                     | ``1675789581``                   |
+|                 | | The response will contain the header ``X-OC-MTime: accepted`` |                                  |
+|                 | | if the mtime was accepted.                                    |                                  |
++-----------------+-----------------------------------------------------------------+----------------------------------+
+| X-OC-CTime      | | Allow to specify a creation time.                             | ``1675789581``                   |
+|                 | | The response will contain the header ``X-OC-CTime: accepted`` |                                  |
+|                 | | if the mtime was accepted.                                    |                                  |
++-----------------+-----------------------------------------------------------------+----------------------------------+
+| OC-Checksum     | Allow to specify a checksum.                                    |                                  |
```

Comment:
Does it mean the server will validate that the uploaded content matches this checksum? At first I didn't get this explanation, so it should be improved. Especially what kind of checksum it needs to get.

Reply:
No idea yet, will update when I have an answer

Reply:
Updated with the bare minimum